### PR TITLE
[Synfig Studio/Windows] Fixed non-working command "Edit image editing in external tool..."

### PR DESCRIPTION
**Pull request by ice0**
Fix https://github.com/synfig/synfig/issues/1909

@morevnaproject need to add `gspawn-win??-helper.exe` and `gspawn-win??-helper-console.exe` to builder.

### CODE REVIEW

File: synfig-studio/src/gui/app.cpp [hunk 1]
```diff
@@ -3731,15 +3687,37 @@ App::window_shortcuts()
...
+		Gtk::MessageDialog dialog(*App::main_window, _("Please set preferred editing tool in \nEdit->Preferences->Editing"), false, Gtk::MESSAGE_ERROR, Gtk::BUTTONS_CLOSE, true);
+		dialog.set_secondary_text(filename);
 		dialog.set_title(_("Error"));
 		dialog.run();
+		return;
 	}
 
+	char buffer[4096];
+#ifdef WIN32
+	::snprintf(buffer, sizeof(buffer), "\"%s\" \"%s\"",App::image_editor_path.c_str(), filename.c_str());
```
Comment (rodolforg) [inline synfig-studio/src/gui/app.cpp:3704]: Why don't you use etl:strprintf?

Comment (ice0) [reply synfig-studio/src/gui/app.cpp:3704]: Agree. This need to be fixed.

Comment (rodolforg) [reply synfig-studio/src/gui/app.cpp:3704]: I don't know if it has that annoying problems with non-latin characters as it is.

**Comment by ice0** [comment]
> indentation :)

Current indents is correct, and I do not touch other parts just to show they have not changed.

**Comment by ice0** [comment]
@rodolforg please approve PR if other parts is ok for you.

**Review by rodolforg — APPROVED** [review]

**Comment by morevnaproject** [comment]
> @morevnaproject need to add `gspawn-win??-helper.exe` and `gspawn-win??-helper-console.exe` to builder.

Done!

https://github.com/morevnaproject/morevna-builds/commit/89acaacad91cb0fb0d9fc3c3e5787397d51eb359

**Pull request merged**